dev-lang/nim/nim-1.6.6-r2.ebuild

```diff
@@ -15,25 +15,22 @@ SRC_URI="
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="~amd64 ~arm ~x86"
-IUSE="debug experimental +readline"
+IUSE="debug experimental"
 RESTRICT="test"  # need to sort out depends and numerous failures
 
-RDEPEND="readline? ( sys-libs/readline:0= )"
-DEPEND="${RDEPEND}"
+BDEPEND="sys-process/parallel"
 # BDEPEND="test? ( net-libs/nodejs )"
 
-PATCHES=( "${FILESDIR}"/${PN}-0.20.0-paths.patch )
+PATCHES=(
+	"${FILESDIR}"/${PN}-0.20.0-paths.patch
+	"${FILESDIR}"/${PN}-1.6.6-csources-flags.patch
+)
 
 _run() {
 	einfo "Running: ${@}"
```

Comment:
edo?

Reply:
no, it also sets PATH

Reply:
... just do PATH=... edo?

Reply:
4x _run == 4x PATH=... edo ...
kinda ugly imo

maybe 
```bash
_run() {
    PATH=... edo ...
}
```